editor/project_manager.cpp

```diff
@@ -473,16 +473,17 @@ class ProjectDialog : public ConfirmationDialog {
 					}
 					PackedStringArray project_features = ProjectSettings::get_required_features();
 					ProjectSettings::CustomMap initial_settings;
+
 					// Be sure to change this code if/when renderers are changed.
-					int renderer_type = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
-					initial_settings["rendering/vulkan/rendering/back_end"] = renderer_type;
-					if (renderer_type == 0) {
-						project_features.push_back("Vulkan Clustered");
-					} else if (renderer_type == 1) {
-						project_features.push_back("Vulkan Mobile");
+					String renderer_type = renderer_button_group->get_pressed_button()->get_meta(SNAME("rendering_method"));
+					initial_settings["rendering/renderer/rendering_method"] = renderer_type;
+
+					if (renderer_type == "forward_plus" || renderer_type == "mobile") {
+						project_features.push_back("Vulkan");
```

Comment:
There is a mismatch between here and project_settings.cpp. As-is, this will add "Vulkan" to the list of features when creating a new project, but this isn't on the list of supported features.